packages/@aws-cdk/aws-apigatewayv2-authorizers/package.json

```diff
@@ -82,13 +82,15 @@
     "@aws-cdk/aws-apigatewayv2": "0.0.0",
     "@aws-cdk/aws-cognito": "0.0.0",
     "@aws-cdk/core": "0.0.0",
+    "@aws-cdk/aws-lambda": "0.0.0",
```

Comment:
We like to keep this alphabetically sorted. Can you follow the same?

Same for the 'peerDependencies' section.